web/src/main/java/se/inera/intyg/intygsadmin/web/stubs/WCIntegratedUnitsServiceStub.java

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright (C) 2019 Inera AB (http://www.inera.se)
+ *
+ * This file is part of sklintyg (https://github.com/sklintyg).
+ *
+ * sklintyg is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * sklintyg is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package se.inera.intyg.intygsadmin.web.stubs;
+
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import org.springframework.context.annotation.Profile;
+import org.springframework.stereotype.Service;
+import se.inera.intyg.intygsadmin.web.controller.dto.IntegratedUnitDTO;
+import se.inera.intyg.intygsadmin.web.service.WCIntegratedUnitsService;
+
+@Profile("dev")
```

Comment:
Borde nog finnas en specifik profil för denna stubbe, typ wc-unit-stub. 
Och den behöver läggas till i listan av profiler för test i openshift.

Reply:
@kesse Vad är fördelen med att ha en separat profil? Jag tycker att dev stämmer med när jag vill använda stubben.

Reply:
Det fungerar bra med dev när man kör lokalt, men som jag skrev så kör vi inte med den profilen i testerna på openshift och där finns det ingen webcert för IA att prata med.
Så den behöver använda stubben.

Reply:
OK

Reply:
Då då lägger jag till profilen till SPRING_PROFILES_ACTIVE i dintyg. Ok?

Reply:
Eller används filerna under openshift i demo och dintyg? @kesse

Reply:
Inser att det låter lite rörigt. Används filerna i repot som ligger under devops/openshift eller ska jag in och ändra i openshift? @kesse

Reply:
Filerna i devops/openshift  används för test och demo. 
Precis i SPRING_PROFILES_ACTIVE